databricks/koalas/frame.py

```diff
@@ -245,6 +245,13 @@ def __init__(self, data=None, index=None, columns=None, dtype=None, copy=False):
             else:
                 super(DataFrame, self).__init__(_InternalFrame(
                     data, data_columns=index.data_columns, index_map=index.index_map))
+        elif isinstance(data, ks.Series):
```

Comment:
I think he meant the docstring here. yea we should fix

Reply:
yup, I meant the docstring.

Reply:
> I think he meant the docstring in this constructor. yea we should fix

Sorry, im not familiar with it. Please can you fix the docstring?

Reply:
Sure, I'll add it later.